src/DynamoCoreWpf/UI/Prompts/ChangeScaleFactorPrompt.xaml.cs

```diff
@@ -43,64 +51,36 @@ void Cancel_Click(object sender, RoutedEventArgs e)
             DialogResult = false;
         }
 
-        void Slider_Changed(object sender, RoutedPropertyChangedEventArgs<double> e)
-        {
-            RefreshHighlight();
-        }
-
-        void UnitComboBox_Changed(object sender, SelectionChangedEventArgs e)
+        private void RefreshDescription(int val)
         {
-            RefreshUnits();
-            RefreshHighlight();
+            this.DescriptionScaleRange.Text =
+                (val == 0) ? Res.ChangeScaleFactorPromptDescriptionContent0 :
```

Comment:
This will then change to `(val == 0) ? String.Format(Res.ChangeScaleFactorPromptDescriptionContent0, ScaleRange.Item1, ScaleRange.Item2) : ...` as explained above.